include/deal.II/fe/fe_interface_values.h

```diff
@@ -187,6 +187,10 @@ namespace FEInterfaceViews
      * function
      * @p interface_dof_index in the quadrature point @p q_point
      * of the component selected by this view.
+     *
+     * @note The name of the function is supposed to be read as "the jump
+     *   (singular) of the values (plural: one or two possible values) of
+     *   the shape function (singular)".
```

Comment:
> the values (plural) of the shape function (singular)

This may be a stupid question, but is this true for both continuous and discontinuous elements?

Reply:
Well, for continuous elements, the two values happen to be the same, and so the jump is zero.

Reply:
And there's one shape function, right? But aren't there two shape functions (two DoFs) when you have discontinuous FEs?

Reply:
It's still one shape function, but in the case of discontinuous elements, it is nonzero in the interior of one cell and zero on all other cells. So it's one shape function (that one indicated by the arguments) that has two values at the interface.

Reply:
At least that's what this operation does: take the jump of one shape function. For DGQ, there happen to be two (or more, in the case of a vertex) shape functions that are defined at the same node locations. But we're only looking at one of them at any given time.

Reply:
Oh, right. Thanks for taking the time to clarify that. As I said, stupid question :-D